phosphor-power-supply/power_supply.cpp

```diff
@@ -499,6 +499,25 @@ void PowerSupply::analyzeMFRFault()
     {
         if (mfrFault < DEGLITCH_LIMIT)
         {
+            if (statusMFR & 0x80)
```

Comment:
Since the meaning of the STATUS_MFR_SPECIFIC bits varies based on the manufacturer, should it check the driver name like PowerSupply::determineMFRFault()?  Otherwise it is likely not the correct behavior for Bonnell PSUs.